data/net/minecraft/util/Graph.mapping

```diff
@@ -0,0 +1,7 @@
+CLASS net/minecraft/util/Graph
+	METHOD depthFirstSearch (Ljava/util/Map;Ljava/util/Set;Ljava/util/Set;Ljava/util/function/Consumer;Ljava/lang/Object;)Z
+		ARG 0 graph
+		ARG 1 visitedVerticies
+		ARG 2 recursionStack
+		ARG 3 action
+		ARG 4 currentVertex
```

Comment:
```suggestion
	METHOD depthFirstSearch (Ljava/util/Map;Ljava/util/Set;Ljava/util/Set;Ljava/util/function/Consumer;Ljava/lang/Object;)Z
		ARG 0 graph
		ARG 1 nonCyclicalNodes
		ARG 2 pathSet
		ARG 3 onNonCyclicalNodeFound
		ARG 4 currentNode
```

This is a method used purely by the feature cycle sorter and it's used for a specific purpose - to identify cycles and (unfortunately) not return them. I have had for a while, a [mapped + commented](https://github.com/alcatrazEscapee/cyanide/blob/517557efb96c6b44282d269a91317ebd648debce/Common/src/main/java/com/alcatrazescapee/cyanide/codec/FeatureCycleDetector.java#L213) version of this method in Cyanide, and would suggest the names inherited from there.

It would also be nice to add some comments to this effect:
```java
/**
 * Detects if a cycle is present in the given graph, via a depth first search, and returns {@code true} if a cycle was found.
 * 
 * @param nonCyclicalNodes Nodes that are verified to have no cycles involving them.
 * @param pathSet The current collection of seen nodes. When invoked not recursively, this should be an empty set.
 * @param onNonCyclicalNodeFound Invoked on each node as we prove that no cycles can be reached starting from this node.
```